shell/browser/browser_process_impl.cc

```diff
@@ -106,12 +107,12 @@ void BrowserProcessImpl::PostEarlyInitialization() {
   OSCrypt::RegisterLocalPrefs(pref_registry.get());
 #endif
 
+  pref_registry->RegisterDictionaryPref(electron::kWindowStates);
+
   in_memory_pref_store_ = base::MakeRefCounted<ValueMapPrefStore>();
   ApplyProxyModeFromCommandLine(in_memory_pref_store());
   prefs_factory.set_command_line_prefs(in_memory_pref_store());
 
-  // Only use a persistent prefs store when cookie encryption is enabled as that
```

Comment:
can you expand on why this comment is removed? Has the underlying logic regarding only using persistent prefs store when cookie encryption is enabled changed?

Reply:
will approve once tests pass and this comment is addressed